tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator/runtimeassemblies.txt

```diff
@@ -1,357 +0,0 @@
-Microsoft.AspNetCore.dll
```

Comment:
I don't think we should delete this file. Having a change history for this would be helpful.

Reply:
If only generated file that is part of the buildoutput is needed, then including generated file as embedded resource when building nuget package should work right? We can have checked in version of this file in tests?

Reply:
We could, this also serves to keep the change history here if you intend to update the process, since in order to pass, the test would need this file to be updated.